static/js/entry/dashboard.js

```diff
@@ -1,11 +1,14 @@
+require('react-hot-loader/patch');
```

Comment:
It would be nice to have this for the other entry points but we can do that in another PR